src/ngMock/angular-mocks.js

```diff
@@ -1341,13 +1341,13 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
       // service the ngMockE2E version can pass through to the real backend
       originalHttpBackend = $delegate.$$originalHttpBackend || $delegate;
 
-  function createResponse(status, data, headers, statusText) {
+  function createResponse(status, data, headers, statusText, xhrStatus) {
     if (angular.isFunction(status)) return status;
 
     return function() {
       return angular.isNumber(status)
-          ? [status, data, headers, statusText]
-          : [200, status, data, headers];
+          ? [status, data, headers, statusText, xhrStatus || (status === 200 ? 'success' : 'error')]
```

Comment:
I just realized `'success'` is confusing. I was thinking it means something different than it actually does. Maybe we should change it to `complete` or something.
Then you should change both lines to `xhrStatus || 'complete'`.